docs/pages/es/overview.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: page
+title: Descripción General
+permalink: es/overview
+---
+# Introducción a Meshery
+El plano de gestión de la malla de servicios adopta, opera y desarrolla sobre diferentes mallas de servicios.
```

Comment:
Service Mesh term should not be translated. So replace 'El plano de gestion de la malla de servicios' por el plano de gestion de service mesh' and so on each time you're using mallas de servicios.

Reply:
Done!